blog-service/2023-06-27-collections.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Sophos Central (Collection)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - collection
+  - app catalog
+  - saas cloud
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We have revised the method of collecting endpoint data from the Sophos Central platform. Previously, endpoint data was combined with alerts and then sent to Sumo Logic. Now we have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source/).
```

Comment:
```suggestion
We've revised the method of collecting endpoint data from the Sophos Central platform. Previously, endpoint data were combined with alerts and sent to Sumo Logic. We have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source/).
```

Reply:
Can we talk about what customers will achieve after implementing this new approach?  Just a benefit they will get form this. Add after "We have adopted a new approach where endpoint and alert data are sent separately to Sumo Logic.

Reply:
Thanks Mega! I have updated the complete release note.